src/components/users/user_row.js

```diff
@@ -33,6 +33,14 @@ class UserRow extends React.Component {
             </div>
           }
         </td>
+        <td>
```

Comment:
Since we added the `STATUS` column, this also adds some info for normal users there.